build_tools/cmake/rebuild.sh

```diff
@@ -56,7 +56,17 @@ CMAKE_ARGS=(
   "-DIREE_TARGET_BACKEND_CUDA=ON"
 )
 
+echo "Configuring"
+echo "-----------"
 "$CMAKE_BIN" "${CMAKE_ARGS[@]?}" "$@" ..
+
+# Build llvm-symbolizer first and make it discoverable, so any future build
+# steps can include symbolized stack traces if there are crashes.
+echo "Building llvm-symbolizer"
+echo "------------------------"
+"$CMAKE_BIN" --build . --target llvm-symbolizer
+export LLVM_SYMBOLIZER_PATH=$PWD/third_party/llvm-project/llvm/bin/llvm-symbolizer
```

Comment:
This works (despite the only affected Kokoro build not finishing logs uploading 😂), but I'd rather not duplicate it across multiple scripts. I'll try to install in our base Dockerfile next, hoping that https://github.com/llvm/llvm-project/blob/main/llvm/tools/llvm-symbolizer/llvm-symbolizer.cpp is stable enough that we can use a prebuilt version instead of building from source with every CI run.